fprime_fpp/fprime_fpp_install.py

```diff
@@ -212,48 +213,90 @@ def install_fpp_via_git(installation_directory: Path):
             )
             sys.exit(-1)
     with tempfile.TemporaryDirectory() as tools_directory:
-        os.chdir(tools_directory)
-        wget(SBT_URL)
-        with tarfile.open(os.path.basename(SBT_URL)) as archive:
-            def is_within_directory(directory, target):
-                
-                abs_directory = os.path.abspath(directory)
-                abs_target = os.path.abspath(target)
-            
-                prefix = os.path.commonprefix([abs_directory, abs_target])
-                
-                return prefix == abs_directory
-            
-            def safe_extract(tar, path=".", members=None, *, numeric_owner=False):
-            
-                for member in tar.getmembers():
-                    member_path = os.path.join(path, member.name)
-                    if not is_within_directory(path, member_path):
-                        raise Exception("Attempted Path Traversal in Tar File")
+        with safe_chdir(tools_directory):
+            # Install SBT into ephemeral tools directory
+            sbt_env = install_sbt(tools_directory)
+            # Install FPP tools in installation_directory
+            install_fpp_via_git(installation_directory, tools_directory, sbt_env)
```

Comment:
This particular call does not use git anymore....should we change the name?

Reply:
Addressed